component-tools/src/test/java/org/talend/sdk/component/tools/SVG2PngTest.java

```diff
@@ -49,4 +51,15 @@ void convert() throws IOException {
         assertTrue(Files.exists(darkExpected));
         assertTrue(Files.exists(lightExpected));
     }
+
+    @Test
+    void exception() {
+        String path = "D:\\works\\qlik";
```

Comment:
When I want to set an error path, I like it to be more “unreal”
```suggestion
        String path = "not/existing/path";
```